src/excmds.ts

```diff
@@ -6145,7 +6145,8 @@ export async function readerurl() {
  */
 //#content
 export async function reader(...args: string[]) {
-    switch(args[0]) {
+    if (window.location.href.includes("static/reader.html#")) return back()
```

Comment:
@hejops do you have any objections to doing this instead? it seems to work fine in my limited testing

Reply:
Oh yeah, that would simulate `readerold` more accurately. Didn't notice there was a `back` function, that's all.